onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4003,37 +4003,20 @@ def aten_native_layer_norm(
     # where D is the dimension of normalized_shape. For example, if normalized_shape is
     # (3, 5) (a 2-dimensional shape), the mean and standard-deviation are computed
     # over the last 2 dimensions of the input (i.e. input.mean((-2, -1))).
+
     axes_list = [-i for i in range(len(normalized_shape), 0, -1)]
-    axes = op.Constant(value_ints=axes_list)
+    start_axis = axes_list[0]
     if not op.OptionalHasElement(weight):
-        weight = op.CastLike(1, input)
+        one = op.Constant(value_float=1.0)
+        weight = op.Expand(one, op.Shape(input, start=start_axis))
```

Comment:
Can it be rewritten with `ConstantOfShape` to keep in one line?

Reply:
I also think it might be possible to just use the scalar constant one (without expanding it), assuming the ONNX LayerNormalization op permits a weight that is broadcastable to the input. Sadly, the documentation of the op in ONNX and ORT do not clarify this. But the ORT implementation seems to allow it, when I tried it.

Reply:
if only use
···
weight = op.Constant(value_float=1.0)
···
will get below error:
----
E   onnxruntime.capi.onnxruntime_pybind11_state.InvalidArgument: [ONNXRuntimeError] : 2 : INVALID_ARGUMENT : Non-zero status code returned while running LayerNormalization node. Name:'' Status Message: Size of X.shape()[axis:] == 6. Size of scale and bias (if provided) must match this. Got scale size of 1 and bias size of 6

Reply:
> Can it be rewritten with `ConstantOfShape` to keep in one line?

```
op.ConstantOfShape(op.Shape(input), value=op.Constant(value_ints=[1]))
```
always failed.